packages/actions-global/src/hooks/use-on-action-error.ts

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
-import handleActionError, {
-  ActionError,
-  DispatchActionError,
-} from '../actions/handle-action-error';
+import handleActionError, { ActionError } from '../actions/handle-action-error';
 
-export default function useOnActionError(error: ActionError) {
-  const dispatch = useDispatch<DispatchActionError>();
```

Comment:
The type should stay. Why did you need to remove it?

Reply:
Can return it for sure. Just wonder, why explicitly putting types in the internal code (which is not a part of a public API) if TS can use its type-inference super-powers to do the job with less typing from my side?

Reply:
I think with this type annotation TS wouldbe be able to infer what the useCallback and dispatch actually returns. Have you checked?

Reply:
@tdeekens just tried doing the following with the changes from this PR (i.e. with dropped typing inside the function body):

```js
const onActionError = useOnActionError();
onActionError('oups');
```

Got this:

<img width="586" alt="Screenshot 2020-02-13 at 22 16 28" src="https://user-images.githubusercontent.com/1228431/74478946-a5228380-4eae-11ea-8d5f-cbefd253dd44.png">

Reply:
All good, the types are correctly inferred. Thanks for double checking!
I dropped the type params from the other hooks as well. https://github.com/commercetools/merchant-center-application-kit/pull/1335/commits/99d69fef251afbd8e8673c67548af1529364e2fb